src/containers/HubiiApiHoc/tests/selectors.test.js

```diff
@@ -74,28 +74,28 @@ describe('makeSelectTransactionsWithInfo', () => {
     expect(transactionsWithInfoSelector(mockedState)).toEqual(expected);
   });
 
-  it('should mark add tx as loading when prices loading', () => {
+  xit('should mark add tx as loading when prices loading', () => {
```

Comment:
Disabled tests intentional? Dead code?

Reply:
Yes, it is intentional. These tests are not necessary anymore after supported the cache for the transactions data. I will remove these tests.